.gitignore

```diff
@@ -0,0 +1 @@
+build_artefacts
```

Comment:
Thoughts on this @patricksnape. We can always go back and rename later. After all, this is only being used locally ATM.